lib/sandthorn/bounded_context.rb

```diff
@@ -0,0 +1,39 @@
+module Sandthorn
+  module BoundedContext
+    module ClassMethods
+      def aggregate_list
```

Comment:
@jesjos What do you think about changing the aggregate_list name to only aggregates?